app/controllers/api/requests_controller.rb

```diff
@@ -15,8 +15,17 @@ def create
   end
 
   def index
-    requests = Request.all
-    render json: { requests: requests.map { |req| Request::IndexSerializer.new(req) } }
+    requests = Request.all.order('id DESC')
+    if current_user
+      reqs = current_user.offers.map { |req| req[:request_id] }
+      binding.pry
+      requests.each do |req|
+        req.serializable_hash.merge!({ offerable: !reqs.include?(req[:id]) })
+      end
+    end
+    json = { requests: requests.map { |req| Request::IndexSerializer.new(req) } }
```

Comment:
why not just use `each_serializer` I commented on this in another PR

Reply:
Doing now, added ams config to use wrapping as default